api/.eslintrc.json

```diff
@@ -4,7 +4,7 @@
         "es2021": true,
         "node": true
     },
-    "extends": ["eslint:recommended", "airbnb-base"],
+    "extends": ["eslint:recommended", "airbnb-base", "prettier"],
```

Comment:
Curiou: what does `prettier` have to do with eslint config?

Reply:
So if prettier is added at the end here, when you use prettier for formatting and it conflicts with airbnb's config, npm run lint won't throw an error about it. You can test it with this part of the code: https://github.com/Researchify/Researchify/pull/167#discussion_r692588868
If you run prettier on the file and run `npm run lint` without prettier in the extends array, it'll complain about formatting. 
@kdav108 figured this out. :)